governance/veANGLE/gauges.md

```diff
@@ -51,26 +49,29 @@ veANGLE stakers can therefore feel confident staking the maximum duration of 4 y
 
 ## Gauge Types
 
-Curve introduced the gauge system for their CRV token emissions. Users lock their veCRV to vote on "weights" of different Curve liquidity pools. The idea with Angle gauges is to incentivize different types liquidity pools useful to the protocol:
+Curve introduced the gauge system for their CRV token emissions. Users lock their veCRV to vote on "weights" of different Curve liquidity pools. The idea with Angle gauges is to incentivize different types of liquidity pools useful to the protocol:
+
+- sanTokens owners
+- Perpetual owners
+- Liquidity pools on exchanges (like on Curve, on Uniswap)
+- Liquidity pools on other chains (like agEUR/USDC on Quickswap on Polygon for instance)
+
+The gauge system could also be used to incentivize different things than staking contracts. For instance, incentives for Convex voters on Curve could be linked to a gauge (in fact an EOA or a multisig) where veANGLE owners decide to send some rewards, and the multisig would be responsible for placing the incentives on Votium.
 
-* agEUR single side staking
-* sanTokens owners
-* Perpetual owners
-* Liquidity pools on exchanges (like on Curve, on Uniswap)
-* Liquidity pools on different chains (like agEUR/USDC on Quickswap on Polygon for instance)
+Overall, there is no restriction on which protocols or pairs can have a gauge weight other than that they should pass the gauge governance vote. The idea is however that all gauges should have a link with the agEUR or the Angle protocol, as the governance token of the protocol is being distributed as incentive.
 
-The gauge system could also be used to incentivize different things than staking contracts. For instance, incentives for Convex voters on Curve could be linked to a gauge (in fact an EOA or a multisig) where veANGLE owners decide to send some rewards and responsible for then placing the incentives on Votium.
+Given that a wide range of contracts could be considered Angle protocol gauges, the system distinguishes several gauge types:
 
-Overall, there is no restriction on which protocols or pairs can have a gauge weight other than that they should pass the gauge governance vote. The idea is however that gauges should all be linked with agEUR stablecoin or protocol.
+- Type 0 Gauges: mainnet staking contracts internal to the Angle Protocol. LPs of these gauges can receive boosted rewards.
+- Type 1 Gauges: corresponding to Angle Perpetual staking contracts. Note that there will be no boosts for veANGLE holders in these contracts.
+- Type 2 Gauges: External staking contracts. This gauge type includes all staking contracts **not on Angle on Ethereum Mainnet**, wether they are on other protocols (Curve or Convex) or on other chains (Polygon or Avalanche).
```

Comment:
Let's keep Type > 2 Gauges and say that there is no gauge like this at the moment, but that basically any new type can be added and that it'll be added for some other chains gauges.

+ typo: wether